src/Identity/UI/ref/Microsoft.AspNetCore.Identity.UI.netcoreapp3.0.cs

```diff
@@ -6,20 +6,15 @@ namespace Microsoft.AspNetCore.Identity
     public static partial class IdentityBuilderUIExtensions
     {
         public static Microsoft.AspNetCore.Identity.IdentityBuilder AddDefaultUI(this Microsoft.AspNetCore.Identity.IdentityBuilder builder) { throw null; }
-        public static Microsoft.AspNetCore.Identity.IdentityBuilder AddDefaultUI(this Microsoft.AspNetCore.Identity.IdentityBuilder builder, Microsoft.AspNetCore.Identity.UI.UIFramework framework) { throw null; }
     }
 }
 namespace Microsoft.AspNetCore.Identity.UI
 {
-    public partial class DefaultUIOptions
```

Comment:
Could we leave these types in and obsolete them with guidance on how to use the correct API instead of removing it? Are there any consequences to leaving it around, but not do anything?

Reply:
We want to get rid of the types as its confusing to keep them around when the mechanics have completely changed.